### PR TITLE
use node 0.12

### PR DESCRIPTION
**Pull request by nightwing**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -319,14 +319,8 @@ vfsextend(){
...
   $NPM cache clean
-  $NPM install sqlite3@2.1.18
+  $NPM install sqlite3@3.0.5
   $NPM install sequelize@2.0.0-beta.0
-  mkdir -p "$C9_DIR"/lib
-  cd "$C9_DIR"/lib
-  $DOWNLOAD https://raw.githubusercontent.com/c9/install/master/packages/sqlite3/linux/sqlite3.tar.gz
-  tar xvfz sqlite3.tar.gz
-  rm sqlite3.tar.gz
-  ln -sf "$C9_DIR"/lib/sqlite3/sqlite3 "$C9_DIR"/bin/sqlite3
```
Comment (nightwing) [inline install.sh]: @javruben do you know why is this needed? Seems to not be used by anything.

Comment (javruben) [inline install.sh]: I don't understand. Which sqlite is used then if not this one?
